.upptimerc.yml

```diff
@@ -15,6 +15,8 @@ sites:
     url: https://www.find-postgraduate-teacher-training.service.gov.uk/
   - name: Publish teacher training courses
     url: https://www.publish-teacher-training-courses.service.gov.uk/
+  - name: Apply for teacher training
+    url: https://www.apply-for-teacher-training.service.gov.uk/
```

Comment:
Is home page ok or shall we use a more involved healthcheck? (as discussed for find and publish)

Reply:
Oops - just merged on phone and saw this! Yes, we should probably use the proper healthcheck endpoint. I'll raise